src/api/db/schema.rb

```diff
@@ -190,6 +190,12 @@
     t.index ["target_project_id"], name: "index_bs_request_actions_on_target_project_id"
   end
 
+  create_table "bs_request_actions_seen_by_users", id: false, charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
+    t.bigint "bs_request_action_id", null: false
+    t.bigint "user_id", null: false
```

Comment:
Those columns should be integer, not bigint. This implies a change in the migration creating this table.